apps/content/repositories/recitation.py

```
@@ -0,0 +1,79 @@
+        return qs
+
+    def get_recitation_asset(self, asset_id: int, publisher_q: Q) -> dict[str, Any] | None:
+        try:
+            asset = self.get_asset_object(asset_id, publisher_q)
+            if not asset:
+                return None
+            return {
+                "id": asset.id,
+                "name": asset.name,
+                "description": asset.description,
+            }
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)